app/components/Siderbar/index.tsx

```diff
@@ -35,7 +35,7 @@ const SideBarStatic = styled.div`
     max-width: 7rem;
     min-height: calc(100vh - ${(props) => props.theme.headerHeight});
     height: auto;
-    background-color: ${colors.primary};
+    background-color: ${colors.lightGreen};
```

Comment:
Why have we changed this ?

Reply:
Tried to make UI more appealing and hence made those changes. I have added screenshots in the PR description for the reference.